include/openspace/scene/scenegraphnode.h

```diff
@@ -187,6 +192,14 @@ class SceneGraphNode : public properties::PropertyOwner {
     glm::dmat4 _modelTransformCached;
     glm::dmat4 _inverseModelTransformCached;
 
+    properties::BoolProperty _computeScreenSpaceValues;
+    properties::IVec2Property _screenSpacePosition;
+    properties::BoolProperty _screenVisibility;
+    properties::DoubleProperty _distFromCamToNode;
+    properties::DoubleProperty _screenSizeRadius;
+    properties::FloatProperty _visibilityDistance;
+    std::chrono::high_resolution_clock::time_point _lastUpdate;
```

Comment:
Is this _lastUpdate only used for the new properties regarding screen space data? In that case, its name is too generic.